src/transformers/agents/agent_types.py

```diff
@@ -125,6 +128,10 @@ def to_raw(self):
             self._raw = Image.open(self._path)
             return self._raw
 
+        if self._tensor is not None:
+            array = self._tensor.cpu().detach().numpy()
+            return Image.fromarray((255 - array * 255).astype(np.uint8))
```

Comment:
I invert the image colors with this `(255 - array*255)`, else the image appears in negative.